### PR TITLE
Option to get cover image from the song

### PR DESCRIPTION
**Pull request by YajanaRao**
Getting image for each song

### CODE REVIEW

**Comment by guoguo12** [comment]
Thanks! What happens when Billboard doesn't have the cover art for a track? And can you add a unit test for this feature?

**Comment by YajanaRao** [comment]
There will be a default track given by billboard. I will add unit tests.

File: billboard.py [hunk 1]
```diff
@@ -69,9 +70,10 @@ class ChartEntry:
         isNew: Whether the track is new to the chart, as a boolean.
     """
 
-    def __init__(self, title, artist, peakPos, lastPos, weeks, rank, isNew):
+    def __init__(self, title, artist, cover, peakPos, lastPos, weeks, rank, isNew):
         self.title = title
         self.artist = artist
+        self.cover = cover
```
Comment (guoguo12) [inline billboard.py]: Can we call this `image`? It looks like they're not always album covers.

File: billboard.py [hunk 2]
```diff
@@ -226,6 +228,18 @@ def fetchEntries(self):
                 message = "Failed to parse title"
                 raise BillboardParseException(message)
 
+            try:
+                coverSoup = entrySoup.find('img',"chart-list-item__image")
```
Comment (guoguo12) [inline billboard.py]: I think this ought to reuse `_ROW_SELECTOR_FORMAT` in some way.

File: billboard.py [hunk 3]
```diff
@@ -226,6 +228,18 @@ def fetchEntries(self):
...
+                if coverSoup.has_attr('data-src'):
```
[the remark this answers is not in the document]
Comment (YajanaRao) [reply billboard.py]: Here I am checking whether there image or not. If there is no default image I am taking the billboard image given in src attribute

File: billboard.py (hunk 2 above)
Comment (YajanaRao) [reply billboard.py]: Unable to re use _ROW_SELECTOR_FORMAT as it uses div.

File: billboard.py (hunk 1 above)
Comment (YajanaRao) [reply billboard.py]: Updated the codebase

**Comment by YajanaRao** [comment]
I am working on unit tests

**Comment by guoguo12** [comment]
Hey! Any update on this? Do you need any help?

**Comment by YajanaRao** [comment]
Sorry man could not complete unit tests.

**Comment by guoguo12** [comment]
Okay, no worries!

**Comment by guoguo12** [comment]
Merged, thanks for contributing!

**Pull request merged**